src/bijectors/named_bijector.jl

```diff
@@ -158,10 +158,10 @@ function forward(cb::NamedComposition, x)
     
     for t in cb.bs[2:end]
         res = forward(t, rv)
```

Comment:
Another one above and here:
```suggestion
        rv, res_logjac = with_logabsdet_jacobian(t, rv)
```